tests/factory_fixtures/lease.py

```diff
@@ -21,11 +19,23 @@ def _lease_attributes(unitNum, tenant, property):
 
 @pytest.fixture
 def create_lease(faker, lease_attributes, create_property, create_tenant):
-    def _create_lease(tenant=None, property=None, unitNum=None):
+    def _create_lease(
+        tenant=None,
+        property=None,
+        unitNum=None,
+        dateTimeStart=None,
+        dateTimeEnd=None,
+    ):
         unitNum = unitNum or faker.building_number()
         tenant = tenant or create_tenant()
         property = property or create_property()
-        lease = LeaseModel(**lease_attributes(unitNum, tenant, property))
+        dateTimeStart = dateTimeStart or faker.date_time_this_decade()
+        dateTimeEnd = dateTimeEnd or faker.date_time_this_decade(
+            before_now=False, after_now=True
+        )
+        lease = LeaseModel(
+            **lease_attributes(unitNum, tenant, property, dateTimeStart, dateTimeEnd)
+        )
         lease.save_to_db()
```

Comment:
We should probably switch this to use the create method. Just something to be aware of. Whenever we modify old code or create new code.. it should use the `create` method so that we can ensure our objects are valid.

Reply:
```suggestion
        lease = LeaseModel.create(
            LeaseSchema,
            lease_attributes(unitNum, tenant, property, dateTimeStart, dateTimeEnd),
        )
```

When I do this, I get the following error:
```
E       werkzeug.exceptions.BadRequest: 400 Bad Request: {'dateTimeEnd': ['Not a valid datetime.'], 'dateTimeStart': ['Not a valid datetime.']}
```

Reply:
What those objects look like:
```
> /Users/gus/dwellinglybackend/tests/factory_fixtures/lease.py(9)_lease_attributes()->{'dateTimeEnd': datetime.date...3, 13, 54, 25), 'dateTimeStart': datetime.date...0, 20, 40, 19), 'occupants': 47, 'propertyID': 1, ...}
```